src/uu/od/src/od.rs

```diff
@@ -286,26 +470,21 @@ impl OdOptions {
 pub fn uumain(args: impl uucore::Args) -> i32 {
     let args = args.collect_str();
 
-    let opts = create_getopts_options();
-
-    let matches = match opts.parse(&args[1..]) {
-        Ok(m) => m,
-        Err(f) => {
-            show_usage_error!("{}", f);
-            return 1;
-        }
-    };
+    let mut clap_opts = create_opts();
+    let clap_matches = clap_opts
+        .clone() // Clone to reuse clap_otps to print help
+        .get_matches_from(args.clone());
 
-    if matches.opt_present("help") {
-        println!("{}", opts.usage(&USAGE));
+    if clap_matches.is_present(options::HELP) {
+        let _ = clap_opts.print_help();
         return 0;
     }
-    if matches.opt_present("version") {
+    if clap_matches.is_present(options::VERSION) {
         println!("{} {}", executable!(), VERSION);
         return 0;
     }
 
-    let od_options = match OdOptions::new(matches, args) {
+    let od_options = match OdOptions::clap_new(clap_matches, args) {
```

Comment:
please do like the other binary :)

Reply:
Done

Reply:
Wait, I'm not 100% clear what you meant. Do you mean changing `clap_new` to just `new`?